TinyMCE.Blazor/wwwroot/tinymce-blazor.js

```diff
@@ -124,25 +143,16 @@ window.tinymceBlazorWrapper = {
     tinyConf.target = el;
     tinyConf._setup = tinyConf.setup;
     tinyConf.setup = (editor) => {
-      editor.on('init', (e) => {
-        // set the initial value on init
-        dotNetRef.invokeMethodAsync('GetValue').then(value => {
-          editor.setContent(value);
-        });
-      });
-      editor.on('change', (e) => {
-        dotNetRef.invokeMethodAsync('OnChange');
-      })
-      editor.on('input', (e) => {
-        dotNetRef.invokeMethodAsync('OnInput');
-      })
-      editor.on('setcontent', (e) => update('text', editor.getContent({ format: 'text' })));
-      editor.on(blazorConf.modelEvents, (e) => {
+      tinyEventHandler.bindEvent(editor, 'init', (e) => dotNetRef.invokeMethodAsync('GetValue').then(value => { editor.setContent(value); }));
+      tinyEventHandler.bindEvent(editor, 'change', (e) => { dotNetRef.invokeMethodAsync('OnChange'); });
+      tinyEventHandler.bindEvent(editor, 'input', (e) => { dotNetRef.invokeMethodAsync('OnInput'); });
+      tinyEventHandler.bindEvent(editor, 'setcontent', (e) => update('text', editor.getContent({ format: 'text' })));
+      tinyEventHandler.bindEvent(editor, blazorConf.modelEvents, (e) => {
```

Comment:
I'm curious why these two files are almost identical?

Reply:
They are identical.

`tinymce-blazor.js` was the original file users needed to include in the `index.html` for `.net5` and earlier version. After `.net6` components' js libraries would be automatically included if they followed the new naming convention `<component name>.lib.module.js`. We didn't wanted to break things for `.net5` users so just added the new one for `.net6` users instead of having `.net5` users update their deployments